app/models/commit_status.rb

```diff
@@ -49,12 +49,20 @@ def expire_cache
 
   private
 
+  # TODO: do non-db lookups in parallel to save time
   # @return [Hash]
   def combined_status
     @combined_status ||= begin
       if @commit
         statuses = [github_combined_status]
-        statuses += [release_status, *only_production_status, *plugin_statuses].compact if @stage
+        if @stage
+          statuses.concat [
+            release_status,
+            only_production_status,
```

Comment:
```suggestion
            *only_production_status,
```
missing * to expand blob?

Reply:
no, it's only a single status so no need to expand